client/src/api/auth.js

```diff
@@ -0,0 +1,14 @@
+export const signApi = (email, password) => {
+  const opts = {
+    method: "POST",
+    body: JSON.stringify({ email: email, password: password }),
+    headers: {
+      "content-type": "application/json"
+    }
+  };
+  return fetch("/auth/login", opts)
+    .then(res => res.json())
+    .then(data => {
+      return data;
+    });
```

Comment:
```suggestion
```
Since the second `then` outputs the same input, we can remove it since it's unnecessary